### PR TITLE
docs:sort filter tree option

### LINKED ISSUE #1: [Filter Tree] Sort filter option
# Header

### PR DESCRIPTION
**Pull request by stephenN24**
Close #1

### CODE REVIEW

File: Sort filter tree option.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# documentation
+
```
Comment (Windflower2497) [inline Sort filter tree option.md:2]: This should be blank
```suggestion
abc
```

File: Sort filter tree option.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # documentation
-
+abc
```
Comment (Windflower2497) [inline Sort filter tree option.md:2]:
```suggestion

```

File: Sort filter tree option.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# documentation
+### ABC
```
Comment (Windflower2497) [inline Sort filter tree option.md:2]:
```suggestion
### ABC
### DEF
```

Comment (stephenN24) [reply Sort filter tree option.md:2]: No we cannot add this.

Comment (Windflower2497) [inline Sort filter tree option.md:2]:
```suggestion
### ABC
### DEF
```

Comment (Windflower2497) [inline Sort filter tree option.md:2]:
```suggestion
### ABC
### DEF
```

**Review by Windflower2497 — APPROVED** [review]

**Pull request merged**